### DISCUSSION TITLE
Heroku

### DISCUSSION THREAD

**Discussion by ghost** [pyload/pyload#3981] [Help]
How to deploy pyload on heroku?

**Comment by Velociraptor85** [comment]
So there is a dockerimage https://github.com/pyload/pyload/issues/3402 and there is this docu https://devcenter.heroku.com/articles/container-registry-and-runtime maybe you could try that ?
edit: https://medium.com/travis-on-docker/how-to-run-dockerized-apps-on-heroku-and-its-pretty-great-76e07e610e22
